src/main/java/io/github/lucasstarsz/fastj/math/Maths.java

```diff
@@ -136,4 +136,17 @@ public static float magnitude(Point p) {
     public static boolean floatEquals(float a, float b) {
         return Math.abs(a - b) < FloatPrecision;
     }
+
+    /**
+     * Returns the value of lerping between x and y, by the interpolation point t.
+     *
+     * @param x The first {@code double} value.
+     * @param y THe second {@code double} value.
+     * @return The lerped value
+     */
+    public static double lerp(double x, double y, double t) {
```

Comment:
Use `float` instead of `double` -- the `Maths` library is for enhancing the quality of life of using float values. An implementation using `double` could be useful, but not just yet for the scope of this project.